TombEngine/Game/misc.cpp

```diff
@@ -17,7 +17,8 @@ CreatureInfo* GetCreatureInfo(ItemInfo* item)
 
 void TargetNearestEntity(ItemInfo* item, CreatureInfo* creature)
 {
-	float bestDistance = FLT_MAX;
```

Comment:
What's the practical difference between using `FLT_MAX` and `INFINITY` in such case?

Reply:
Done by @Sezzary it's probably the same i think.

Reply:
Since `FLT_MAX` is technically a """valid""" distance, `INFINITY` > `FLT_MAX`, and our compiler seems to be okay with using the macro, I'd say it's the cleaner way in such cases.